lib/bruter.ts

```diff
@@ -97,9 +94,6 @@ export default function runSolver(
         const solutions = brute(pattern, matrix, true);
         return solutions.map((solution) => ({ match, solution }));
       })
-      // it's possible that a sequence was found which includes skips
-      // filter out solutions that are longer than the buffer size!
-      .filter((seq) => seq.solution.length <= bufferSize)
```

Comment:
This one is still needed as the bruter can skip the first digit in the matrix even if it is not present in the sequence.

https://github.com/cxcorp/cyberpunk2077-hacking-solver/blob/1e724760ba3d4984e02666a62db469f42584b0fe/lib/bruter.ts#L340-L350

Reply:
To clarify, this means that if the bruter is fed a sequence of length 8, it's possible for it to return a solution of length 9.

Reply:
Understood. Let me add some testing around this as well before I proceed 😄